src/variables/_code.scss

```diff
@@ -0,0 +1,3 @@
+$white: #ffffff;
```

Comment:
why this vars in a separate file not in variables.scss?

Reply:
i have removed that file